lib/wallaby/dsl.ex

```diff
@@ -1,5 +1,26 @@
 defmodule Wallaby.DSL do
-  @moduledoc false
+  @moduledoc """
+  Sets up the Wallaby DSL in a module.
+
+  All functions in `Wallaby.Browser` are now accessible without a module name
+  and `Wallaby.Browser`, `Wallaby.Element` and `Wallaby.Query` are all aliased.
+
+  ## Example
+
+  ```
+  defmodule MyPageObject do
+    use Wallaby.DSL
+
+    def register(session) do
+      session
+      |> visit("/registration.html")
+      |> fill_in(Query.text_field("Name"), with: "Chris")
+      |> fill_in(Query.text_field("email"), with: "c@keathly.io")
+      |> click(Query.button("Save"))
```

Comment:
One other addition that would be good to add here is to move these queries into module attributes or functions. Thats a pattern that I think we should be actively trying to encourage. What do y'all think?